Client/Frontend/Home/FirefoxHomeViewController.swift

```diff
@@ -753,18 +753,38 @@ extension FirefoxHomeViewController: DataObserverDelegate {
 extension FirefoxHomeViewController {
     @objc func openBookmarks() {
         homePanelDelegate?.homePanelDidRequestToOpenLibrary(panel: .bookmarks)
+        TelemetryWrapper.recordEvent(category: .action,
+                                     method: .tap,
+                                     object: .firefoxHomepage,
+                                     value: .yourLibrarySection,
+                                     extras: [TelemetryWrapper.EventObject.libraryPanel.rawValue: TelemetryWrapper.EventValue.bookmarksPanel.rawValue])
```

Comment:
Am I just being stupid by not understanding why we need rawvalue here?

Reply:
Extras are of type `[String: Any]`. I used these so that we're not using actual strings which are prone to typos. I guess I could pass the value without the rawValue and cast it in the telemetry file, to be fair.